Dockerfile

```diff
@@ -1,51 +1,17 @@
-FROM redis:6-buster
```

Comment:
I'm not exactly sure how I feel about moving this to an individual service. I was talking with the initial implementer of the Dockerfiles awhile back about it, and we thought it would be best to leave it in as an ease of use setup, for those who wish to use Docker. I'm still not entirely sold on this and swapping it over due to the fact that the whole Docker setup was seen more as a "temporary" setup anyways, that somehow became more and more prevalent over time. I may need to sit on this for awhile.

Reply:
The reason for moving redis to separate service is to get rid of the installation of packages that are not in the redis image, but are needed by the project. Also setting up redis as separate service is super simple. The fact that redis is not even a requirement for running the project suggests to me that it should not be the base for the image. Also it seems to be a standard in the docker world to only have one service per image.

Reply:
Ahhh yes, I understand with the whole docker standard of running one per image. Okay, well if this can be changed over and included in the docker-compose file, and maybe some additional instruction within the README, I would definitely be okay moving forward with this. Just ensure that redis has a password included as well in the config. I know with the latest redis-clients in the other PR, redis will actually crash the app if a password is supplied when none exists on the redis instance. So long as we still have the password working (and the `travis.yml` stuff is updated as well) as intended, I don't see a problem moving this forward then :)